pkg/apis/core/types.go

```diff
@@ -2174,6 +2176,7 @@ type ContainerStatus struct {
 	ImageID      string
 	// +optional
 	ContainerID string
+	Started     bool
```

Comment:
If this is optional, it needs to be a pointer.  We can't add non-optional fields, ergo this should be a pointer.  Otherwise an old client will not set it and it will be false, which is super confusing.

Reply:
But... actually this isn't set by any client AFAIK, nor unmarshaled, this is just an internal object for the kubelet.
Making it a pointer only calls for NPE, IMHO.